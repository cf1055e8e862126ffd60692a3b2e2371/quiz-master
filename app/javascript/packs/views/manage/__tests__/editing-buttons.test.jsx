import React from 'react'
import renderer from 'react-test-renderer'
import EditingButtons from '../editing-buttons'

describe('#render', () => {
  it('should show snapshot', () => {
    const component = renderer.create(
      <EditingButtons onSend={() => {}} onCancel={() => {}} />,
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
