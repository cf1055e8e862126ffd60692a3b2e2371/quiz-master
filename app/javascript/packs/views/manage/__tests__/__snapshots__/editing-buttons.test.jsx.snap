// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render should show snapshot 1`] = `
<div>
  <button
    className="btn btn-primary"
    onClick={undefined}
  >
    Send
  </button>
  <button
    className="btn btn-default"
    onClick={undefined}
  >
    Cancel
  </button>
</div>
`;
